syntax = "proto3";

import "google/protobuf/wrappers.proto";

service DiamondService {
   rpc FilterInit (google.protobuf.BoolValue) returns (FilterRole);
}

message FilterRole {
  bool sendExamples = 1;
  bool requestExport = 2;
}
